### PR TITLE
Adds slips to the scoreboard

### PR DESCRIPTION
**Pull request by MadmanMartian**
A silly little idea, where the scoreboard counts how many times somebody slipped during the round.

### CODE REVIEW

**Comment by kilozombie** [comment]
The only thing Vince Vagner was ever truly successful at

File: code/game/gamemodes/scoreboard.dm [hunk 1]
```diff
@@ -438,6 +438,7 @@
 		dat += "<B>Station Deficit:</B> [num2text(profit,50)]<BR>"}*/
 	dat += {"<B>Food Eaten:</b> [score["foodeaten"]]<BR>
 	<B>Times a Clown was Abused:</B> [score["clownabuse"]]<BR>
+	<B>Number of slips: </B> [score["slips"]]<BR>
```
Comment (Exxion) [inline code/game/gamemodes/scoreboard.dm]: This should probably be more specific, and should definitely be title case like (almost) all of the others

Comment (MadmanMartian) [reply code/game/gamemodes/scoreboard.dm]: Rightio

File: code/game/gamemodes/scoreboard.dm [hunk 2]
```diff
@@ -438,6 +438,7 @@
 		dat += "<B>Station Deficit:</B> [num2text(profit,50)]<BR>"}*/
 	dat += {"<B>Food Eaten:</b> [score["foodeaten"]]<BR>
 	<B>Times a Clown was Abused:</B> [score["clownabuse"]]<BR>
+	<B>Number of Times Someeone was Slipped: </B> [score["slips"]]<BR>
```
Comment (Exxion) [inline code/game/gamemodes/scoreboard.dm]: >Someeone

**Comment by mtpowell** [comment]
Is it possible to add how many slips a clown got?

**Comment by gbasood** [comment]
@mtpowell it'd be somewhat insane trying to track how every spill was directly clown-related

**Comment by mtpowell** [comment]
@gbasood Couldn't we just add a slip counter to the clowns PDA ?

**Comment by Exxion** [comment]
Sure, if you want to record specifically PDA slips

**Comment by gbasood** [comment]
And miss literally all the other slips that clowns do, because the PDA isn't the most common source of slips.

**Pull request merged**